vm.go

```diff
@@ -458,7 +551,9 @@ func (vm *vm) try(ctx1 context.Context, f func()) (ex *Exception) {
 				iterTail := vm.iterStack[iterLen:]
 				for i := range iterTail {
 					if iter := iterTail[i].iter; iter != nil {
-						returnIter(iter)
+						vm.try(vm.ctx, func() {
+							returnIter(iter)
+						})
```

Comment:
Lower down we have:

```
if x1.ExportType().Kind() == reflect.String {
					ex.ignoreStack = true
				}
				v := x1.baseObject(vm.r)
				if v != nil {
					if v.__wrapped != nil {
						if nErr, ok := v.__wrapped.(error); ok {
							ex.nativeErr = nErr
						}
					}
}
```

but `x1` can be null in certain tests and the returned value from `ExportType()` would be `nil` which would then cause a `panic`.